ideas/58-mainnet.md

```diff
@@ -0,0 +1,71 @@
+## Preamble
+
+    Idea: 58-mainnet-default
+    Title: Use Mainnet as the default network
+    Status: In progress
+    Created: 2017-12-08
+
+
+## Summary
+Alpha builds of Status should default to Mainnet. We should be using Status in a real-world environment.
+
+## Swarm Participants
+
+- Lead Contributor: @adambabik
+- Testing & Evaluation: <!-- @username -->
+- Contributor: @themue
+- Contributor: <!-- @username -->
+- PM: @chadyj 
+- UX (if relevant): <!-- @username -->
+
+## Product Overview
+
+Currently the Status alpha builds use testnet, but using Mainnet is a key milestone for the public launch of Status. There will be potentially be major performance and security implications so these issues should be surfaced as soon as possible so the app can be tested in a real-world environment. 
+
+From a users perspective, Mainnet should be used as it is required by most popular DApps, and is needed for real transactions, so the full potential of Status can't be seen until we switch.
+
+## Product Description
+
+This is a default setting change, so there aren't any user facing options.
+
+Some users, such as DApp creators may want to test their app on testnet (or some other use case), so they will be able to toggle networks as can be done currently.
+
+## Requirements & Dependancies
+
+Security and performance should should be discussed and considered.
+
+Also, several Dapps should be tested. 
```

Comment:
We have to make both more concrete. Would appreciate a discussion about it (criteria for security, performance, and reliability as well as a selection of DApps for repeatable tests).